### PR TITLE
Ranks on match histories

### PR DESCRIPTION
**Pull request by Sciguymjm**

### CODE REVIEW

File: webapp/src/Components/Player/Overview/MatchHistory/OverviewMatchHistoryRow.tsx [hunk 1]
```diff
@@ -70,6 +70,19 @@ class OverviewMatchHistoryRowComponent extends React.PureComponent<Props> {
         const replayGameMode = replay.gameMode
         const replayScore = <ColouredGameScore replay={replay}/>
         const replayResult = getReplayResult(replay, player)
+
+        const averageRank = Math.round(replay.ranks.reduce((previous, current, idx) => previous + current)
```
Comment (twobackfromtheend) [inline webapp/src/Components/Player/Overview/MatchHistory/OverviewMatchHistoryRow.tsx]: How is unranked handled? I think it should be left out and not be added as a 0.

File: webapp/src/Components/Player/Overview/MatchHistory/OverviewMatchHistoryRow.tsx [hunk 2]
```diff
@@ -101,11 +114,16 @@ class OverviewMatchHistoryRowComponent extends React.PureComponent<Props> {
                                     {replayScore}
                                 </Typography>
                             </Grid>
-                            <Grid item xs={2}>
+                            <Grid item xs={1}>
```
Comment (twobackfromtheend) [inline webapp/src/Components/Player/Overview/MatchHistory/OverviewMatchHistoryRow.tsx]: How does this look on mobile

File: webapp/src/Components/ReplaysSearch/ReplayDisplayRow.tsx [hunk 3]
```diff
@@ -134,6 +148,9 @@ class ReplayDisplayRowComponent extends React.PureComponent<Props> {
                         <ColouredGameScore replay={replay}/>
                     </Typography>
                 </Grid>
+                {aboveSm && (<Grid item xs={1} className={classes.listGridItem}>
```
Comment (twobackfromtheend) [inline webapp/src/Components/ReplaysSearch/ReplayDisplayRow.tsx]: Move the <Grid to the next line so <Grid> and </Grid> are aligned.

File: webapp/src/Components/Player/Overview/MatchHistory/OverviewMatchHistoryRow.tsx (hunk 2 above)
Comment (Sciguymjm) [reply webapp/src/Components/Player/Overview/MatchHistory/OverviewMatchHistoryRow.tsx]: https://github.com/SaltieRL/DistributedReplays/pull/320/files/323d678cdc3c5a9a324fc226a1cdbb1344b74b95#diff-a1b0c89b5849680028a2a48469fbd18dR97

Comment (twobackfromtheend) [reply webapp/src/Components/Player/Overview/MatchHistory/OverviewMatchHistoryRow.tsx]: ???

Comment (twobackfromtheend) [reply webapp/src/Components/Player/Overview/MatchHistory/OverviewMatchHistoryRow.tsx]: ah github is dumb

Comment (twobackfromtheend) [reply webapp/src/Components/Player/Overview/MatchHistory/OverviewMatchHistoryRow.tsx]: How does it look at mininum sm width then

Comment (Sciguymjm) [reply webapp/src/Components/Player/Overview/MatchHistory/OverviewMatchHistoryRow.tsx]: great

**Pull request merged**